homeassistant/components/fritzbox_callmonitor/sensor.py

```diff
@@ -163,7 +213,7 @@ def __init__(self, host, port, sensor):
 
     def connect(self):
         """Connect to the Fritz!Box."""
-        _LOGGER.debug("Setting up socket...")
+        LOGGER.debug("Setting up socket...")
         self.sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
```

Comment:
This should be moved out to the PyPI package.

Reply:
Discussion in https://github.com/home-assistant/core/pull/40736#discussion_r505333417